python/cuml/utils/input_utils.py

```diff
@@ -16,61 +16,50 @@
 
 import cuml.utils.numba_utils
 
+import copy
 import cudf
 import cupy as cp
 import numpy as np
-import rmm
 import warnings
 
 from collections import namedtuple
 from collections.abc import Collection
+
 from cuml.utils import rmm_cupy_ary
 from numba import cuda
 
 
+cuml_array = namedtuple('cuml_array', 'array n_rows n_cols dtype')
+
+# inp_array is deprecated and will be dropped
```

Comment:
Does this have an issue we can reference with any other detasils about the deprecation? (planned drop version, dependent events upon which is can be removed?)